### PR TITLE
SDK-865: Added documentation for REST api

### PR DESCRIPTION
**Pull request by Kyrylo-Meretskyi**
[SDK-865](https://spryker.atlassian.net/browse/SDK-865)

### CODE REVIEW

File: docs/conventions.md [hunk 1]
```diff
@@ -89,6 +89,16 @@ No conventions yet
 - Contract __MUST__ exist only in case existing logic provides for an extension by the user.
 - Contract is a public API and __MUST__ follow [Spryker plugin interfaces specification](https://spryker.atlassian.net/wiki/spaces/RFC/pages/1038092073/INTEGRATED+RFC+Plugin+interface+specification).
 
+## REST API
+- Controller __SHOULD__ have no business logic.
+- Controller __SHOULD__ be named by template of task's id.
+  For task `validation:php:codestyle` - controller name is `ValidationController` and controller's action name is `validationPhpCodestyle` (task's full id in camelCase).
```
Comment (pavelmaksimov25) [inline docs/conventions.md:95]: For example, for the task...
The controller name should be `ValidatePhpCodestyleController`. We need to keep it more specific a bit. 
action name can be `index()` in this case, so the resulting route should be `POST /api/v1/validate-php-codestyle`

File: docs/conventions.md [hunk 2]
```diff
@@ -89,6 +89,16 @@ No conventions yet
...
+- Controller __MUST__ be placed in namespace `SprykerSdk\Sdk\Presentation\RestApi\Controller\v1`. The `v1` is current version of SDK API.
```
Comment (pavelmaksimov25) [inline docs/conventions.md:96]: According to the doc provided by architect, we should use `v1.0` for versioning.  Could you discuss that with Andrea please and update the convention respectively?

File: docs/conventions.md [hunk 3]
```diff
@@ -89,6 +89,16 @@ No conventions yet
...
+- Route __MUST__ be placed in `src/Presentation/RestApi/Resources/routing.yaml`
+- Route __SHOULD__ be named by template `api_${version}_${controller_name}_${action_name}`.
```
Comment (pavelmaksimov25) [inline docs/conventions.md:98]: This is how the route automatically generates by Symfony. Please, change this statement according to my comment above.

**Review by pavelmaksimov25 — CHANGES_REQUESTED** [review]

**Comment by pavelmaksimov25** [comment]
Ticket number missed

File: docs/conventions.md (hunk 3 above)
Comment (Kyrylo-Meretskyi) [reply docs/conventions.md]: Fixed

File: docs/conventions.md (hunk 1 above)
Comment (Kyrylo-Meretskyi) [reply docs/conventions.md]: Fixed

**Review by pavelmaksimov25 — APPROVED** [review]

**Pull request merged**